components/datadog/apps/tracegen/images/tracegen/main.go

```diff
@@ -0,0 +1,103 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"math"
+	"sync/atomic"
+	"time"
+
+	"golang.org/x/time/rate"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+)
+
+var tracecount atomic.Uint32
+var spancount atomic.Uint32
+
+func reportStats() {
+	for {
+		time.Sleep(5 * time.Second)
+		tc := tracecount.Swap(0)
+		sc := spancount.Swap(0)
+		fmt.Printf("Finished %d traces/s, %d spans/second.\n", tc/5, sc/5)
+	}
+}
+
+func main() {
+	tps := flag.Float64("tps", 1, "Target number of traces to generate per second.")
+	spt := flag.Uint("spt", 2, "Number of spans to put in each trace. (>=1)")
+	testDuration := flag.Duration("testtime", 0, "Amount of time to run the test. A value of '0' means the test will continue indefinitely.")
```

Comment:
Ideally we should allow the tests to control these flags using env vars, it would be much easier than changing the cmd flags which might depend on the infra (docker/ecs/k8s).

Reply:
Added in https://github.com/DataDog/test-infra-definitions/pull/573/commits/3c02c4c909014e4fb59ca1705bc298fa7ec7b85b